### DISCUSSION TITLE
pytorch BCEWithLogitsLoss equivalence

### DISCUSSION THREAD

**Discussion by wangjiawen2013** [tracel-ai/burn#2651] [Q&A] [answered]
Hi,
pytorch has `BCEWithLogitsLoss`, is there the burn equivalence ?

**Comment by laggui** [comment, marked as answer]
Yep, you can use the `BinaryCrossEntropyLoss` with logits:

```rust
let loss = BinaryCrossEntropyLossConfig::new().with_logits(true).init(&device);
```
